plexapi/library.py

```diff
@@ -2849,6 +2861,133 @@ def _loadData(self, data):
+
+            Parameters:
+                after (obj): :class:`~plexapi.library.ManagedHub` object to move the item after in the collection.
+
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: When trying to move a Hub that is not a Managed Recommendation.
+        """
+        if not self._promoted:
+            raise BadRequest('Collection must be a Managed Recommendation to be moved')
+        key = f'/hubs/sections/{self.librarySectionID}/manage/{self.identifier}/move'
+        if after:
+            key = f'{key}?after={after.identifier}'
+        self._server.query(key, method=self._server._session.put)
+
+    def remove(self):
+        """ Removes a managed hub from the library's Managed Recommendations.
+
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: When trying to remove a Hub that is not a Managed Recommendation
+                    or when the Hub cannot be removed.
+        """
+        if not self._promoted:
+            raise BadRequest('Collection must be a Managed Recommendation to be removed')
+        if not self.deletable:
+            raise BadRequest(f'{self.title} managed hub cannot be removed' % self.title)
+        key = f'/hubs/sections/{self.librarySectionID}/manage/{self.identifier}'
+        self._server.query(key, method=self._server._session.delete)
+
+    def updateVisibility(self, recommended=None, home=None, shared=None):
+        """ Update the managed hub's visibility settings.
+
+            Parameters:
+                recommended (bool): True to make visible on your Library Recommended page. False to hide. Default None.
+                home (bool): True to make visible on your Home page. False to hide. Default None.
+                shared (bool): True to make visible on your Friends' Home page. False to hide. Default None.
+
+            Example:
+
+                .. code-block:: python
+
+                    managedHub.updateVisibility(recommended=True, home=True, shared=False).reload()
+                    # or using chained methods
+                    managedHub.promoteRecommended().promoteHome().demoteShared().reload()
+        """
+        params = {}
+        if recommended is not None:
+            params['promotedToRecommended'] = int(recommended)
+        if home is not None:
+            params['promotedToOwnHome'] = int(home)
+        if shared is not None:
+            params['promotedToSharedHome'] = int(shared)
+
+        if not self._promoted:
+            params['metadataItemId'] = self.identifier.rsplit('.')[-1]
+            key = f'/hubs/sections/{self.librarySectionID}/manage'
+            self._server.query(key, method=self._server._session.post, params=params)
+        else:
+            key = f'/hubs/sections/{self.librarySectionID}/manage/{self.identifier}'
+            self._server.query(key, method=self._server._session.put, params=params)
+        return self
```

Comment:
Need to automatically reload so that the current attribute values get updated.
```suggestion
        return self.reload()
```